src/vm/ceeload.cpp

```diff
@@ -526,6 +526,13 @@ void Module::Initialize(AllocMemTracker *pamTracker, LPCWSTR szName)
 
     m_dwTransientFlags &= ~((DWORD)CLASSES_FREED);  // Set flag indicating LookupMaps are now in a consistent and destructable state
 
+#ifdef FEATURE_COLLECTIBLE_TYPES
```

Comment:
Moving to the beginning of the function so that the ReadyToRunInfo::Initialize can check Module::IsCollectible